reana_client/utils.py

```diff
@@ -65,15 +67,15 @@ def yadage_load(workflow_file, toplevel='.'):
     }
 
     try:
-        return yadageschemas.load(spec=workflow_file, specopts=specopts,
-                                  validopts=validopts, validate=True)
+        yadageschemas.load(spec=workflow_file, specopts=specopts,
+                           validopts=validopts, validate=True)
 
     except ValidationError as e:
         e.message = str(e)
         raise e
 
 
-def cwl_load(workflow_file):
+def cwl_load(workflow_file, **kwargs):
```

Comment:
Also here.

Reply:
Same here. For CWL workflows, when passing `TARGET: foo` for example, it will end up here.